rpcserver.go

```diff
@@ -987,6 +988,23 @@ func (r *rpcServer) NewAddr(ctx context.Context,
 
 	var err error
 
+	// Parse the proof courier address if one was provided, otherwise use
+	// the default specified in the config.
+	var proofCourierAddr *url.URL
+
+	if r.cfg.DefaultProofCourierAddr != nil {
```

Comment:
When would this be `nil`? I think we should be able to rely on this being set, as we have the fallback parsing in the `server.go` file. So we can just do: `proofCourierAddr := r.cfg.DefaultProofCourierAddr`.